modules/ldsg.js

```diff
@@ -4,6 +4,39 @@ const Augur = require("augurbot-ts"),
   Discord = require("discord.js"),
   /** @type {string[]} */
   banned = require("../data/banned.json").features.suggestions;
+const hasLink = /http(s)?:\/\/(\w+(-\w+)*\.)+\w+/,
+  affiliateLinks = {
```

Comment:
Small whitespace tweak
```suggestion

const hasLink = /http(s)?:\/\/(\w+(-\w+)*\.)+\w+/;
const affiliateLinks = {
```